packages/components/src/components/SquiggleViewer/ViewerProvider.tsx

```diff
@@ -415,6 +419,7 @@ export const InnerViewerProvider = forwardRef<SquiggleViewerHandle, Props>(
       <ViewerContext.Provider
         value={{
           rootValue: _rootValue,
+          visibleRootPath,
```

Comment:
Before we were sort of using ``rootValue`` for this, but it was pretty confusing imo. So now, we use ``rootValue`` to always be the Path Root (no edges), then have an optional ``visibleRootPath`` to indicate a View Override - the root that it will show.